python/kwiver/sprokit/processes/pytorch/utils/models.py

```diff
@@ -222,47 +222,35 @@ def __init__(self, app_model='', motion_model='', interaction_model='', bbox_mod
 
         self.model_list = model_list
 
-        if RnnType.Appearance in self.model_list:
+        def config_model(model, model_path):
+            """Move model to GPU if use_gpu_flag is true and initialize the model
+            from model_path if truthy.
+
+            """
             if use_gpu_flag:
-                self.appearance = AppearanceLSTM().cuda()
-            else:
-                self.appearance = AppearanceLSTM()
-            if app_model:
-                snapshot = torch.load(app_model)
-                self.appearance.load_state_dict(snapshot['state_dict'])
+                model = model.cuda()
```

Comment:
Using `.cuda` makes a lot of assumptions. This should at least be `.to(device)` even if device is hard-coded to 0. What would be even better is if the device was able to be specified.